core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -1442,7 +1442,8 @@ short ExExeUtilHBaseBulkLoadTcb::work()
            ComCondition *cond;
            Lng32 entryNumber;
            while ((cond = diagsArea->findCondition(EXE_ERROR_ROWS_FOUND, &entryNumber)) != NULL) {
-              errorRowCount = cond->getOptionalInteger(0);
+              if (errorRowCount < cond->getOptionalInteger(0))
```

Comment:
Actually, I do have one question: You mention that each warning is for a different range. Should we add the rowcounts instead of using the max?